src/libertem/analysis/com.py

```diff
@@ -7,11 +7,38 @@
 from .base import AnalysisResult, AnalysisResultSet
 from .masks import BaseMasksAnalysis
 from libertem.corrections.coordinates import rotate_deg, flip_y, identity
-
+from .helper import GeneratorHelper
 
 log = logging.getLogger(__name__)
 
 
+class ComTemplate(GeneratorHelper):
+
+    short_name = "com"
+    api = "create_com_analysis"
+
+    def __init__(self, params):
+        self.params = params
+
+    def get_docs(self):
+        docs = ["# COM Analysis",
+                "***about com analysis ***"]
+        return '\n'.join(docs)
+
+    def convert_params(self):
+        params = [f'dataset=ds']
+        for k in ['cx', 'cy']:
+            params.append(f'{k}={self.params[k]}')
+        params.append(f"mask_radius={self.params['r']}")
+        return ', '.join(params)
+
+    def get_plot(self):
+        # any other better way to plot ?
+        plot = ["plt.figure()",
+                "plt.imshow(com_result['intensity'].data / com_result['intensity'].data.max())"]
```

Comment:
Is this division necessary? `imshow` should already normalize, right? For center of mass, there are a few more interesting plots you can do for the results, like `curl`, `divergence` and a field visualization; possibly could add some of them here, too.

Reply:
![no_image](https://user-images.githubusercontent.com/59721339/85286859-05ef4780-b4b1-11ea-95d7-6db29e8d7441.png)

>
>possibly could add some of them here, too.

Okay, then I have to switch to `ctx.run()`

Reply:
> then I have to switch to ctx.run()

Or use the appropriate post-processing functions

Reply:
Regarding the plot, what shape does `com_result['intensity']` have? And what dtype?

Reply:
`<BufferWrapper kind=nav dtype=float32 extra_shape=(3,)>`

Reply:
Okay, that means you have something like `(256, 256, 3)` (assuming a signal shape of `(256, 256)` - if you just put that unchanged into `imshow`, it will be interpreted as RGB data, which is not correct (and the reason for the strange error message). There needs to be some post-processing code here!

Reply:
> There needs to be some post-processing code here!

... or work towards the `get_results` UDF method: #628